src/main/java/org/team1540/base/power/PowerManager.java

```diff
@@ -122,23 +151,44 @@ private void stopScaling() {
   }
 
 
+  /**
+   * Determines if the current is currently spiking. If power limiting is not engaged,
+   * returns pdp.getTotalCurrent() &gt; currentSpikePeak.
+   * If power limiting is engaged, returns pdp.getTotalCurrent() &gt; currentTarget - currentMargin.
+   *
+   * @return Boolean representing if the current is spiking.
+   */
+  public boolean currentIsSpiking() {
```

Comment:
Refactor to `isCurrentSpiking()` in accordance with [styleguide](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).